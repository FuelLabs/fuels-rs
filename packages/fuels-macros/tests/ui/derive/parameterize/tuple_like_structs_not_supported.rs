use fuels_macros::Parameterize;

#[derive(Parameterize)]
struct SomeStruct(pub u64, pub String);

fn main() {}
