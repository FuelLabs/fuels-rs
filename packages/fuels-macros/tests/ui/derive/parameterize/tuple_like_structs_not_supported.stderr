error: Tuple-like structs not supported
 --> tests/ui/derive/parameterize/tuple_like_structs_not_supported.rs:4:19
  |
4 | struct SomeStruct(pub u64, pub String);
  |                   ^^^^^^^^^^^^^^^^^^^
